ShopifySharp/Services/DraftOrder/DraftOrderService.cs

```diff
@@ -17,61 +17,29 @@ public class DraftOrderService : ShopifyService
         /// <param name="shopAccessToken">An API access token for the shop.</param>
         public DraftOrderService(string myShopifyUrl, string shopAccessToken) : base(myShopifyUrl, shopAccessToken) { }        
 
-        private async Task<int> _CountAsync(ICountFilter filter = null)
-        {
-            var req = PrepareRequest("draft_orders/count.json");
-            
-            if (filter != null)
-            {
-                req.QueryParams.AddRange(filter.ToQueryParameters());
-            }
-            
-            var response = await ExecuteRequestAsync<int>(req, HttpMethod.Get, rootElement: "count");
-
-            return response.Result;
-        }
-
-        /// <summary>
-        /// Retrieves a count of the shop's draft orders. 
-        /// </summary>
-        /// <param name="filter">Options for filtering the count.</param>
-        public virtual async Task<int> CountAsync(ICountFilter filter = null)
```

Comment:
We probably still want a method that accepts a generic filter, just like the list methods accept a generic ListFilter.